libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx

```diff
@@ -1,22 +1,17 @@
-/* eslint-disable @nrwl/nx/enforce-module-boundaries */
 import React from 'react';
-import { Icon, IconType, Row } from '@inheartive/ui/atoms';
-import TextStyled from './TextStyled';
+import { Icon, IconType, Row, Text } from '@inheartive/ui/atoms';
 import { colors } from '@inheartive/ui/theme';
 interface IAuctionHeartsProps {
   quantity: number;
-  testID?: string | undefined;
 }
 
 function AuctionHearts(props: IAuctionHeartsProps) {
-  const { quantity, testID } = props;
+  const { quantity } = props;
 
   return (
     <Row space={2} alignItems='center'>
-      <Icon name={IconType.favoriteOutline} size={35} color={colors.secondary[600]} />
-      <TextStyled fontSize='xl' testID={testID}>
-        {quantity}
-      </TextStyled>
+      <Text>{quantity}</Text>
+      <Icon name={IconType.favoriteOutline} size={30} color={colors.secondary[600]} />
```

Comment:
Why did you delete testID ?

- [ ] Explain

Reply:
1. There was no added value in test since we're checking if component renders quantity
2. Prop title was misleading because it suggests that it is AuctionHearts test id - but it was somewhere deeper - assigned to a text